<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reglog</class>
 <widget class="QWidget" name="reglog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>reglog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius: 30px;
opacity: 0.5;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLineEdit" name="logNameEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="logPswEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logExitButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string> 注册</string>
    </attribute>
    <widget class="QPushButton" name="regButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="regExitButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="regNameEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="regPswEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="reregPswEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>重复密码</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
